runipy/main.py

```diff
@@ -114,7 +119,12 @@ def main():
         profile_dir = None
 
     logging.info('Reading notebook %s', payload.name)
-    nb = read(payload, 'json')
+    try:
+        # Ipython 3
+        nb = read(payload, '3')
```

Comment:
Should be an `int`. ( https://ipython.org/ipython-doc/3/api/generated/IPython.nbformat.html#IPython.nbformat.reads ) Probably good to be consistent, as well.